console/my-project/quickstart.mdx

```diff
@@ -80,94 +73,26 @@ On Windows, you can use the third-party application [PuTTYgen](https://www.chiar
 5. Complete the steps on the screen to finish:
     - Fill in the **Key-comment** field with a name to help you identify this key pair
     - Click the **Save public key** button and save it in the folder of your choice
-    - Click the **Save private key** button and save it in the same folder
+    - Click **Save private key** and save it in the same folder
     - Select the content of the public key (the sequence of characters under "Public key for pasting into OpenSSH authorized_keys file"). Copy it, as you will need this in the next step.
 
-### How to upload the public SSH key to your Scaleway project
+### How to upload the public SSH key to your Scaleway Project
 
-You must upload the content of the public part of the SSH key pair you just created to the Scaleway interface. This is then associated with your project, and can be transferred to your Instances during the boot process.
+You must upload the content of the public part of the SSH key pair you just created to the Scaleway interface. This is then associated with your Project, and can be transferred to your Instances during the boot process.
 
-1. From the [Scaleway console](https://console.scaleway.com), navigate to the [Credentials](/console/my-project/concepts/#credentials) tab of your Project Dashboard.
+1. From the [Scaleway console](https://console.scaleway.com), navigate to the [Credentials](/identity-and-access-management/iam/concepts/#api-key) tab of your Project Dashboard.
```

Comment:
I think the link here needs to go to a #credentials concept somewhere (we should put one in My Project) since it's about SSH keys, not API keys?

Reply:
link to the SSH key concept